test/Store.test.js

```diff
@@ -227,6 +228,7 @@ describe('Store', function() {
       spy.calledOnce.should.eql(true);
       spy.alwaysCalledWith('', {
```

Comment:
Hi Vinay,

I watched your video message, and I took a deeper look into the tests as they are.  This test already spys `sendMessage` and ensures that the message is sent with the `portName`:

> it('should send a message with the correct dispatch type and payload given an extensionId',)

I think that I need to test that `wrapStore` behaves correctly instead?   The missing change that is not tested is that listener `dispatchResponse` created by `wrapStore` should discard messages that do not include its port.

However I do not see any `wrapStore` tests at all!   Shall I try to create some?